homu/main.py

```diff
@@ -1779,6 +1818,10 @@ def main():
         db_query(db, 'SELECT treeclosed_src FROM repos LIMIT 0')
     except sqlite3.OperationalError:
         db_query(db, 'ALTER TABLE repos ADD COLUMN treeclosed_src TEXT')
+    try:
+        db_query(db, 'SELECT squash FROM pull LIMIT 0')
+    except sqlite3.OperationalError:
+        db_query(db, 'ALTER TABLE pull ADD COLUMN squash TEXT')
```

Comment:
Is there a reason why this is stored as text instead of integer?

Reply:
No, I wasn't careful enough when copy-pasting from the `treeclosed_src` migration. I used an `INT` in the `CREATE TABLE`.